baggers.js

```diff
@@ -2990,85 +2990,53 @@ var data = {
           "title": "Agilité, Scrum & Kanban:",
           "abstract": "Introduction aux valeurs et principes de base de l'agilité, et aux fondamentaux de Scrum.<br/>- Présentation ou discussions.<br/>- Découverte ou approfondissements",
           "tags": [
-            "Java",
-            "Jenkins",
-            "BDD",
-            "Big Data",
             "Agile",
             "Scrum",
-            "Kanban",
-            "Flink",
-            "Intégration continue",
-            "Test",
-            "Tests unitaires",
-            "Usine logicielle",
-            "Industrialisation"
+            "Kanban"
           ],
           "lang": [
             "fr"
           ]
         },
         {
-          "title": "Apache Flink : la 4eme Génération de Processing est la:",
-          "abstract": "Introduction à Flink.<br/>- Présentation ou discussions.<br/>- Découverte ou approfondissements",
+          "title": "Devops:",
+          "abstract": "Introduction à Devops.<br/>- Présentation ou discussions.<br/>- Découverte ou approfondissements",
           "tags": [
-            "Java",
-            "Jenkins",
-            "BDD",
-            "Big Data",
-            "Agile",
-            "Scrum",
-            "Kanban",
-            "Flink",
-            "Intégration continue",
-            "Test",
-            "Tests unitaires",
-            "Usine logicielle",
+            "Docker",
+			"Jenkins",
+			"Monitoring",
+			"Kubernetes",
             "Industrialisation"
           ],
           "lang": [
             "fr"
           ]
         },
         {
-          "title": "Devops & Integration Continue :",
-          "abstract": "Outils et méthodes pour les équipes de développement qui veulent réussir dans l'agilité<br/>- Présentation & discussions",
+          "title": "Kubernetes:",
```

Comment:
Same as previous. Why this `:` character which seems to tell that you wanted to add something.